documentation/puppetserver_vs_passenger.markdown

```diff
@@ -4,6 +4,10 @@ title: "Puppet Server: Puppet Server vs. Apache/Passenger Puppet Master"
 canonical: "/puppetserver/latest/puppetserver_vs_passenger.html"
 ---
 
+> ## Important: Deprecation warning
+>
+> The apache/passenger Puppet master is deprecated and will be removed in a future Puppet release.
+
```

Comment:
Just to make sure, it's not already gone as of Puppet 6? Just want to make sure docs around it aren't only relevant for upgrades.

Reply:
I think Webrick support and the `puppet master` command were removed in PUP-8544 and PUP-8591.

It doesn't look like the docs for it were removed from Puppet 6: https://puppet.com/docs/puppet/6.0/services_master_webrick.html

PUP-9055 is a complication.

Reply:
Yeah, apache/passenger masters were deprecated in Puppet 4 and finally removed in 6. This file is no longer applicable.